File: solutions/1329-minimum-cost-to-move-chips-to-the-same-position/solution.js
/**
 * @param {number[]} position
 * @return {number}
 */
var minCostToMoveChips = function(position) {
    let e=0,o=0;
    for(let i=0;i<position.length;i++)
    {
        if((position[i]&1))
            o++;
        else
            e++;
    }
    return o<e?o:e;
};

File: solutions/1329-minimum-cost-to-move-chips-to-the-same-position/solution.c
int minCostToMoveChips(int* position, int positionSize) {
    int e=0,o=0;
    for(int i=0;i<positionSize;i++)
    {
        if((*(position+i)&1))
            o++;
        else
            e++;
    }
    return o<e?o:e;
}
